docbook/src/docbkx/chapters/appendix-diy.xml

```diff
@@ -0,0 +1,1309 @@
+                    </listitem>
+                    <listitem>
+                        <para>Create an image of the configured server. Auto Scale will use this
+                            server image to scale up or down. </para>
+                    </listitem>
+                    <listitem>
+                        <para>Create a load balancer, modify it to have the Least-Connections
+                            algorithm, and attach the Cloud Server image you created as a node.
+                        </para>
+                    </listitem>
+                    <listitem>
+                        <para> Configure the Auto Scale scaling group, as well as the scaling
+                            policies for scaling up and down; in the response you will get the
+                            webhooks that Cloud Monitoring will use in notifications (you create the
+                            notifications next and then attach them to your notification plan).
+                            Note: a “webhook” is a URL that triggers an action, such as sending an
+                            email. The webhooks the scaling group creates allow monitoring
+                            notifications that trigger the scaling policies. </para>
+                    </listitem>
+                    <listitem>
+                        <para> Create notifications for scaling up and scaling down using the
+                            scaling group webhooks, and update the notification plan that you
+                            specified in the monitoring check.  </para>
+                    </listitem>
+                    <listitem>
+                        <para> Run a load test using the Apache Benchmarking tool and observe
+                            notifications and Auto Scale: Critical notification and scale-up;
+                            without load: OK notification and scale down.</para>
+                    </listitem>
+                    <listitem>
+                        <para>Tear down the deployment.</para>
+                    </listitem>
+                </orderedlist>
+            </para>
+            <para>
+                <tip>
+                    <para>
+                        <itemizedlist>
+                            <listitem>
+                                <para>Use a text editor, such as Sublime, to form the commands. Copy
+                                    the command from this document into the editor and make the
+                                    required changes there. Then, paste the results into a shell
+                                    window and execute the command (press Enter). Note: Sublime
+                                    isn’t free, but there are several editors out there like
+                                    Notepad++, Vim, and Brackets, that are free. Trying to use the
+                                    commands, even modified, from the Word file directly may not
```

Comment:
This whole paragraph seems a bit awkward, especially the stuff about Sublime Text being non-free, but the reference to "Word" in particular is wrong, since this documentation is not in the form of a Word file. I would personally leave out the entire paragraph, but if you want to include something like this, here's what I would say:

"Copying and pasting commands directly from this documentation into your terminal program may be somewhat accident-prone, so we recommend you first copy them into your text editor of choice, modify them as required there, and then copy them into your terminal."